content/docs/introduction/glossary.md

```diff
@@ -71,6 +71,25 @@ The [Pushgateway](../../instrumenting/pushing/) persists the most recent push
 of metrics from batch jobs. This allows Prometheus to scrape their metrics
 after they have terminated.
 
+### Remote Read
+
+Remote read is a Prometheus feature that allows transparent reading time series from
+other systems (such as long term storage) as part of queries.
+
+### Remote Read Adapter
+
+Not all systems directly support remote read. A remote read adapter sits between
+Prometheus and another system, converting time series requests and responses between them.
+
+### Remote Read Endpoint
+
+A remote read endpoint is what Prometheus talks to when doing a remote read.
+
+### Remote Write
```

Comment:
Aren't there also the same "Adapter" and "Endpoint" variants for "Remote Write"?